examples/proxy/record-encryption/cluster-ip/README.md

```diff
@@ -1,54 +1,63 @@
+   configuration for Vault or AWS, depending on your KMS provider:
+
+2. Deploy the Example:
    ```sh
    oc apply -k cluster-ip
    ```
 
 # Try out the example
 
-1. Create a key for topic `trades` in the KMS:
-   ```
+1. Create a key for topic `trades` using the instructions applicable to your KMS provider:
+
+   Vault:
+   ```sh
    vault write -f transit/keys/KEK_trades
    ```
-2. Create a topic `trades` on the cluster, via the proxy:
+   AWS:
+   ```sh
+   aws kms create-alias --alias-name alias/KEK_trades --target-key-id $(aws kms create-key | jq -r '.KeyMetadata.KeyId')
    ```
-   oc run -n proxy -qi proxy-producer --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-topics.sh --bootstrap-server proxy-service:9092 --create -topic trades
+2. Create a topic `trades` on the cluster, via the proxy:
+   ```sh
+   oc run -n kafka-proxy -qi create-topic --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-topics.sh --bootstrap-server proxy-service:9092 --create -topic trades
    ```
 3. Produce some messages to the topic:
+   ```sh
+   echo 'IBM:100\nAPPLE:99' | oc run -n kafka-proxy -qi proxy-producer --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server proxy-service:9092 --topic trades
    ```
-   echo 'IBM:100\nAPPLE:99' | oc run -n proxy -qi proxy-producer --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- bin/kafka-console-producer.sh --bootstrap-server proxy-service:9092 --topic trades
-   ```
-4. Consume messages *direct* from the Kafka Cluster, showing that they are encrypted.
-   ```
+4. Consume messages *direct* from the Kafka Cluster, showing that they are encrypted:
+   ```sh
     oc run -n kafka cluster-consumer -qi --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- ./bin/kafka-console-consumer.sh  --bootstrap-server my-cluster-kafka-bootstrap:9092 --topic trades --from-beginning --timeout-ms 10000
    ```
-5. Consume messages from the *proxy* showing they get decrypted automatically.   
-   ```
-    oc run -n proxy proxy-consumer -qi --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- ./bin/kafka-console-consumer.sh  --bootstrap-server proxy-service:9092 --topic trades --from-beginning --timeout-ms 10000
+5. Consume messages from the *proxy* showing they get decrypted automatically:
+   ```sh
+    oc run -n kafka-proxy proxy-consumer -qi --image=registry.redhat.io/amq-streams/kafka-37-rhel9:2.7.0 --rm=true --restart=Never -- ./bin/kafka-console-consumer.sh  --bootstrap-server proxy-service:9092 --topic trades --from-beginning --timeout-ms 10000
    ```   
 
 # Cleaning up
 
 When you have finished with this example, you can remove it from the OpenShift Cluster like this:
 
 ```sh
-oc apply -d cluster-ip
+oc delete -k cluster-ip
```

Comment:
out of curiosity why the switch from `apply -d` to `delete -k` ?

Reply:
It was just plain wrong before.  The '-k' means kustomize.  Apply applies things, it doesn't delete.